sleepecg/io/ecg_readers.py

```diff
@@ -42,10 +43,32 @@ class ECGRecord:
     lead: Optional[str] = None
     id: Optional[str] = None
 
+    def export(self, filename: Union[str, Path]) -> None:
+        """
+        Export to CSV.
+
+        Parameters
+        ----------
+        filename : str | pathlib.Path
+            File name to write to.
+        """
+        export_ecg_record(self, filename)
+
+    def plot(self, **kwargs: np.ndarray) -> None:
+        """
+        Plot ECG record.
+
+        Parameters
+        ----------
+        **kwargs : np.ndarray
+            Additional annotations to be plotted with different markers.
+        """
```

Comment:
One last thing, I think `ECGRecord.plot`'s docstring should give the same information as `plot_ecg`'s:

```suggestion
    def plot(self, **kwargs: np.ndarray):
        """
        Plot ECG time series with optional markers.

        Parameters
        ----------
        **kwargs : np.ndarray
            Positions of annotations (i.e. heartbeats) in samples. If more than one marker
            sequence is given, the keywords will be used as labels in the plot legend.

        Returns
        -------
        fig : matplotlib.figure.Figure
            The figure.
        ax : matplotlib.axes.Axes
            The axes in the figure.
        """
```